content/en/docs/getting-started/_index.md

```diff
@@ -61,73 +60,115 @@ olm-operator       1/1     1            1           5m52s
+### Installing via SDK for development purpose
 
-```sh 
-$ kubectl get packagemanifest -n olm
-NAME                               CATALOG               AGE
-cassandra-operator                 Community Operators   26m
-etcd                               Community Operators   26m
-postgres-operator                  Community Operators   26m
-prometheus                         Community Operators   26m
-wildfly                            Community Operators   26m
-```
+[Operator-SDK][operator-sdk] provides a feature which allows you to install easily OLM for development purposes. 
 
-To install the etcd operator in the default namespace, first create an `OperatorGroup` for the default namespace: 
+Check the [SDK intallation guide][sdk-installation-guide]. Then, see that you are able to install OLM on your cluster by running `operator-sdk olm install` and then, you also can uninstall it with `operator-sdk olm uninstall`. For more info about how to integrate your project using [Operator-SDK][operator-sdk] CLI tool see teh section [OLM integration][sdk-olm-integration].  
 
-```sh
-$ cat operatorgroup.yaml
-kind: OperatorGroup
-apiVersion: operators.coreos.com/v1
-metadata:
-  name: og-single
-  namespace: default
-spec:
-  targetNamespaces:
-  - default
-
-$ kubectl apply -f operatorgroup.yaml
-operatorgroup.operators.coreos.com/og-single created
+### Installing an operator bundle via SDK for development purpose
+
+You can use the [Operator-SDK][operator-sdk] CLI to run your bundle locally with `operator-sdk run bundle`.
+
+To learn more about packaging your operator for OLM, installing/uninstalling an operator etc, visit the [Core Tasks](/docs/tasks/) and the [Advanced Tasks](/docs/advanced-tasks/) section of this site.
+
+
+### From the source
+
+## Run locally with minikube
+
+This command starts minikube, builds the OLM containers locally with the minikube-provided docker, and uses the local configuration in [local-values.yaml][local-values.yaml] to build localized deployment resources for OLM.
+
+```bash
+# To install and run locally
+$ make run-local
 ```
 
-Then create a subscription for the etcd operator: 
-
-```sh 
-$ cat subscription.yaml
-apiVersion: operators.coreos.com/v1alpha1
-kind: Subscription
-metadata:
-  name: etcd
-  namespace: default
-spec:
-  channel: singlenamespace-alpha
-  installPlanApproval: Automatic
-  name: etcd
-  source: operatorhubio-catalog
-  sourceNamespace: olm
-  startingCSV: etcdoperator.v0.9.2
-
-$ kubectl apply -f subscription.yaml
-subscription.operators.coreos.com/etcd created
+You can verify that the OLM components have been successfully deployed by running `kubectl -n local get deployments`
+
+**NOTE** It is recommended for development purposes and will use the source locally.
+
+## Customizing OLM installation 
+
+Deployments of OLM can be stamped out with different configurations by writing a `values.yaml` file and running commands to generate resources.
```

Comment:
Just copy and paste. The goal is we move forward with https://github.com/operator-framework/operator-lifecycle-manager/pull/2046

So, it adds the OLM info from the current installation guide to centralize here.

Reply:
Maybe changing it to "Default values used while installing OLM with `operator-sdk olm install` can be overridden by providing the custom configurations in a `values.yaml` file, and then using that file to generate the new manifests for deploying custom configured OLM." might help?

Reply:
It has no relation with the `operator-sdk olm install`. The `operator-sdk olm install` will not get in count it. 
this info is in the install guide https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/install/install.md#customizing-olm-installation which is used in the README in th Getting started of the OLM README see : https://github.com/operator-framework/operator-lifecycle-manager#getting-started.

The idea of this PR is to move the Getting Started instructions from the OLM README and add them in the Getting Started of the olm docs. So that, we can link the OLM docs webside in the readme and replace that by the Getting Started here.